tests/phpunit/api/v4/Entity/ConformanceTest.php

```diff
@@ -85,10 +85,6 @@ public function tearDown(): void {
    * @throws \CRM_Core_Exception
    */
   public function getEntitiesHitech(): array {
-    // Ensure all components are enabled so their entities show up
-    foreach (array_keys(\CRM_Core_Component::getComponents()) as $component) {
-      \CRM_Core_BAO_ConfigSetting::enableComponent($component);
-    }
```

Comment:
This was redundant with the `setUp` function in this class which calls `enableAllComponents()`